python/pythonlab/unittest_runner/tests/sample_main.py

```diff
@@ -1,2 +1,13 @@
 # sample main for testing
+from neighborhood import Painter
+
 print("Hello world")
+p1 = Painter(0,1)
```

Comment:
should we make a second sample file for a sample painter main so the two tests for stdout and painter don't have to handle both cases?

Reply:
I intentionally had both within the same file to check that if a content editor includes both `print` and neighborhood function calls, that the validation protocol still works as intended.
It seems likely that a user's program could include both?

Reply:
that's fair, we can leave as-is!